components/tools/OmeroPy/test/integration/clitest/cli.py

```diff
@@ -21,6 +21,7 @@
 
 
 import pytest
+import warnings
```

Comment:
Rather than warning, this file could likely just import the testlib names as aliases.